liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/PackageServerMojo.java

```diff
@@ -120,20 +123,33 @@ protected void doExecute() throws Exception {
     }
 
     private void validateInclude() throws MojoFailureException {
+        ArrayList<String> includeValues;
+        List<String> includeStrings;
+
+        if (include != null && !include.isEmpty()) {
+            include = include.trim();
+            includeStrings = Arrays.asList(include.split(","));
+            includeValues = new ArrayList<String>(includeStrings);
+            for (int i = 0; i < includeValues.size(); i++) {
+                String value = includeValues.get(i);
+                includeValues.set(i, value.trim());
+            }
+        } else {
+            includeValues = new ArrayList<String>();
+        }
+
         // if jar, validate include options, and add runnable
         if (packageType.equals("jar")) {
-            if (include == null) {
-                include = "runnable";
-            } else {
-                if (include.contains("usr") || include.contains("wlp")) {
-                    throw new MojoFailureException("Package type jar cannot be used with `usr` or `wlp`.");
-                }
-                
-                if (!include.contains("runnable")) {
-                    include.concat(",runnable");
-                }
+            if (includeValues.contains("usr") || includeValues.contains("wlp")) {
+                throw new MojoFailureException("Package type jar cannot be used with `usr` or `wlp`.");
+            }
+
+            if (!includeValues.contains("runnable")) {
+                includeValues.add("runnable");
             }
         }
+
+        include = String.join(",", includeValues);
```

Comment:
I don't like that this will return an empty String, whereas previously the include value could still be null. The following code is affected by include not being null I think.
```
private String getPackageFileType() throws MojoFailureException {
    	if (packageType != null && packageType.equals("jar")) {
            if (include == null || include.equals("all") || include.equals("minify")) {
                return ".jar";
            } else {
                throw new MojoFailureException("The jar packageType requires `all` or `minify` in the `include` parameter");
            }
    	} else {
            if (packageType != null && !packageType.equals("zip")) {
                log.info(packageType + " not supported. Defaulting to 'zip'");
            }
            packageType = "zip";
            return ".zip";
        }
    }```

Reply:
`getPackageFileType()` runs before `validateInclude()`, so this line wouldn't affect it, but I'll add a check for the `includeValues` size - `include` will be untouched if there are no values. 

However, the pom.xml could potentially set `include` to empty string so I'll add a check there.